server/src/main/java/org/elasticsearch/cluster/service/ClusterApplierService.java

```diff
@@ -446,7 +446,8 @@ private void applyChanges(UpdateTask task, ClusterState previousClusterState, Cl
         if (nodesDelta.hasChanges() && logger.isInfoEnabled()) {
             String summary = nodesDelta.shortSummary();
             if (summary.length() > 0) {
-                logger.info("{}, reason: {}", summary, task.source);
+                logger.info("{}, term: {}, version: {}, reason: {}",
```

Comment:
should we update the same message in MasterService?

Reply:
I think the ClusterApplierService one is more useful since it logs on all nodes and also when adding a no-master block, but it could be useful in the MasterService too so I pushed a02075dcb55.